src/03-control-flow.js

```diff
@@ -14,7 +14,11 @@
  *  isEqual(10, "10");
  *  //> false
  */
-function isEqual(a, b) {}
+function isEqual(a, b) {
+  if (a === b) {
+    return true;
+  } else return false;
```

Comment:
In situations like line 20 while it is not incorrect to write an ` else ` statement we actually do not need one.  In the event that ` a === b ` returns ` true ` the function will end its run.  So it is safe to say that ` if a !== b ` we can simply return ` false` as there are no other options.